src/components/LiquidityMining/Create/Steps/RewardAmount/index.tsx

```diff
@@ -24,75 +17,68 @@ const FlexWrapper = styled(Flex)`
   `}
 `
 interface RewardAmountProps {
-  rewardsObject: RewardsObject
-  setRewardsObject: React.Dispatch<Actions>
+  rewardsArray: Reward[]
```

Comment:
I find it odd to have Array on the name, just a suggestion that might be better

```suggestion
  rewardsList: Reward[]
```

Reply:
Well List is also data structure. And this is difference between list and array according to google 🤓 

> List is used to collect items that usually consist of elements of multiple data types. An array is also a vital component that collects several items of the same data type.

I need a life xD